### ISSUE TITLE
Which pytorch version ?

### ISSUE THREAD

**Issue by tzrtzr000** [ShuangXieIrene/ssds.pytorch#37]

**Comment by blueardour** [comment]
It seems the author employ version before 0.4 when setting up the project. Many features after 0.4 are changed.  such as the variable, the nogradient clause

**Comment by cvtower** [comment]
> It seems the author employ version before 0.4 when setting up the project. Many features after 0.4 are changed. such as the variable, the nogradient clause

Yes, test result(rfb_lite_mobilenetv2_train_voc && fssd_lite_mobilenetv2_train_voc.yml) are almost all zero.(0.4.0)

**Comment by cvtower** [comment]
Here, i also tried the yolo_v3_mobilenet_v2_voc model with pytorch 0.3.0, all result is zero.

**Comment by blueardour** [comment]
my test on mobilnet_v2_yolo_v2_voc gives a mAP of 0.721956. 
Check my result at https://github.com/ShuangXieIrene/ssds.pytorch/issues/41
